resources/lang/en/categories.php

```diff
@@ -7,4 +7,7 @@
     'title' => 'Title',
     'list' => 'List',
     'total_book' => 'Total Book',
+    'options' => 'Options',
+    'confirm_deletion' => 'Confirm deletion!',
+    'are_you_sure_you_want_to_delete_category' => 'Are you sure you want to delete category',
```

Comment:
Dùng text như ri nhé, truyền thêm tên nữa á. `Are you sure to delete this category: :name? `